shared/lib/constants.py

```diff
@@ -281,8 +291,8 @@
             "per capita",
             "percapita",
             "per person",
-            # remove "rate" or "rates" if is not followed by certain words (used as one metric)
-            f"\brate(s)?\b(?!\s*{_RATE_WORDS_TO_SKIP}\s+rate(s)?)",
+            # remove "rate" or "rates" if is not preceded by certain words (used as one metric)
+            f"rates?{_RATE_WORDS_LOOK_BEHINDS}",
```

Comment:
Why no `\b`?  Also, why no `(s)?`?

Reply:
no `\b` because these words are used in classifier and removing stop words and:
- it was having trouble getting recognized in the heuristic classifier when used with the additional regex logic there: https://github.com/datacommonsorg/website/blob/ec4b48e9d6746241e8db791bd04ee13d64456a4e/server/lib/nl/detection/heuristic_classifiers.py#L270
- when removing stop words, the logic there already adds '\b' anyways: https://github.com/datacommonsorg/website/blob/ec4b48e9d6746241e8db791bd04ee13d64456a4e/shared/lib/utils.py#L79

updated `(s)?` to just `s?` because it's the same thing, only difference is whether or not `s` is in a capturing group and there's no need for `s` to be a capture group